museum_export/on_EmbARK_server/EmbARK_Web_Kiosk/Templates/API/marble.json

```diff
@@ -35,6 +35,10 @@
 					{"authority": "IA", "term": "<!--#4DTEXT [Keywords]Word-->", "uri": "http://vocab.getty.edu/page/ia/<!--#4DTEXT [Keywords]_AAT_SubjectiD-->"}
 				<!--#4Dif (Selected Record Number([Keywords])< Records in Selection([Keywords])) -->,<!--#4Dendif-->
```

Comment:
This is actually the template file used to translate things from the 4D database in Embark, it isn't really a json file.  The crazy syntax is what directs the 4D database to pull out and process fields.

This is really confusing, because the file is required to have a .json extension to be able to be processed by the EmbARK json translator, even though the file is not json.

Reply:
what a riddle. what's a json file, but not a json file.